pkg/btc-wallet/src/js/components/lib/send.js

```diff
@@ -68,30 +69,39 @@ const Send = ({ stopSending, value, conversion }) => {
   const checkPayee = (e) => {
     setError('');
 
-    let payeeReceived = e.target.value;
-    let isPatp = ob.isValidPatp(payeeReceived);
-    let isAddress = validate(payeeReceived);
-    if (isPatp) {
-      let command = { 'check-payee': payeeReceived };
+    const validPatPCommand = (validPatP) => {
+      let command = { 'check-payee': validPatP };
       api.btcWalletCommand(command);
       setTimeout(() => {
         setCheckingPatp(false);
       }, 5000);
       setCheckingPatp(true);
       setPayeeType('ship');
-      setPayee(payeeReceived);
+      setPayee(validPatP);
+    };
+
+    let payeeReceived = e.target.value;
+    let isPatp = ob.isValidPatp(payeeReceived);
```

Comment:
This isPatp variable should be called on the deSigged payeeReceived so that we don't need the unnecessary if/else branch.

Reply:
...that makes more sense, ha. Just pushed the fix.